services/frontend/src/components/experience/ExperienceView.jsx

```diff
@@ -11,9 +12,9 @@ const ExperienceView = ({ experienceInfo }) => {
       return (
         <div className="bodyStyle">
           {lineStrings.map((line, index) => (
-            <>
+            <Fragment key={line}>
```

Comment:
is it ok for the key to be a string with spaces? what if it is really long? I feel like it wont be efficient in terms of memory

Reply:
According to my research, the length does not impact performance and there are examples on the antd website that have keys with spaces

Reply:
okay!